library_generation/test/resources/output/gapic-generator-java-pom-parent-2.24.0.pom

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this is a test resource, could you remove all unused part?

Reply:
My bad, it slipped into my commits!

Reply:
Removed